test/clj-tests.lfe

```diff
@@ -693,3 +693,11 @@
 (deftest dec
   (is-match 2 (clj:dec 3))
   (is-match 4.0 (clj:dec 5.0)))
+
+(deftest str
+  (is-equal () (clj:str))
+  (is-equal "123" (clj:str 1 2 3))
+  (is-equal "abc" (clj:str "a" "b" "c"))
+  (is-equal "abc" (clj:str 'a 'b "c"))
+  (is-equal "1a2b" (clj:str 1 'a 2 "b"))
+  (is-equal "2.00c" (clj:str 2.0 'c)))
```

Comment:
Can we add a test or two with a `let` and confirm that `str`-ing a variable works as desired too?

Also, for cases like this where the expected values are literals, I think it's better to use `is-match` so we get more descriptive errors.